seqr/views/utils/terra_api_utils.py

```diff
@@ -152,6 +153,7 @@ def user_get_workspace_access_level(user, workspace_namespace, workspace_name):
 
     try:
         r = _user_anvil_call('get', path, user)
+    # TerraNotFoundException is handled to allow seqr continue working when Terra is not available
```

Comment:
when Terra is not available it wouldn't be returning a 404 though...

Reply:
The exception handling is required only when the Terra API is not available in the case the API server is down. Once a time, I experienced that anvil `dev` API down caused the user couldn't show the local projects.

Reply:
Ah I see. Can you update the comment to add something like "once local access is deprecated this should be taken out" or something like that